agent/probes.go

```diff
@@ -114,6 +115,12 @@ func NewTopologyProbeBundleFromConfig(g *graph.Graph, hostNode *graph.Node) (*pr
 				return nil, fmt.Errorf("Failed to initialize Libvirt probe: %s", err)
 			}
 			probes[t] = libvirt
+		case "sriov":
```

Comment:
can you activate it in the tests even if we don't have srvi-o on CI it will test that it starts properly
https://github.com/skydive-project/skydive/blob/master/tests/tests.go#L91

Reply:
added